tests/adapters/references/test_references.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from optimade.adapters import Reference
+from optimade.models import ReferenceResource
+
+
+class TestReference:
+    """Test Reference adapter"""
+
+    def test_instantiate(self, RAW_REFERENCES):
+        """Try instantiating Reference for all raw test references"""
+        for reference in RAW_REFERENCES:
+            new_Reference = Reference(reference)
+            assert isinstance(new_Reference.entry, ReferenceResource)
+
+    def test_setting_entry(self, capfd, RAW_REFERENCES):
```

Comment:
TIL some new pytest magic :)

Reply:
[Fixtures](https://docs.pytest.org/en/latest/fixture.html) for the win :)